testdata/exec_fail.ivy

```diff
@@ -47,3 +47,9 @@ x
 # index must be integer
 1 2[1 2 log 2]
 	X
+
+1 / 0
+	X
+
+1 / 2 2 rho 0
```

Comment:
not sure these tests trigger it. i did

(100 100 rho 1) / (100 100 rho 0)

and checked that the new code ran.

Reply:
It's true that `1 / 2 2 rho 0` doesn't trigger it in ordinary ivy, but it does in the test.
'go test' runs with maximal parallelism, via value.MaxParallelismForTesting.